@navikt/core/css/darkside/internalheader.darkside.css

```diff
@@ -68,38 +66,28 @@ button.navds-internalheader__title {
   display: flex;
   align-items: center;
   justify-self: center;
-  gap: var(--a-spacing-2);
-  border-left: 1px solid var(--ac-internalheader-divider, var(--a-gray-600));
-  color: var(--ac-internalheader-text, var(--a-text-on-inverted));
-}
-
-a.navds-internalheader__title:hover,
-button.navds-internalheader__title:hover,
-.navds-internalheader__button:hover {
-  background-color: var(--ac-internalheader-hover-bg, var(--a-surface-inverted-hover));
-}
+  gap: var(--ax-spacing-2);
+  border-left: 1px solid var(--ax-border-subtleA);
+  color: var(--ax-text-default);
 
-.navds-internalheader__title:focus-visible,
-.navds-internalheader__button:focus-visible {
-  box-shadow: var(--navds-internalheader-focus);
-  outline: none;
+  &:hover {
+    background: var(--ax-bg-neutral-moderate-hover);
+  }
 }
 
-@supports not selector(:focus-visible) {
-  .navds-internalheader__title:focus,
-  .navds-internalheader__button:focus {
-    box-shadow: var(--navds-internalheader-focus);
-    outline: none;
+.navds-internalheader__title,
+.navds-internalheader__button {
+  &:focus-visible {
+    outline: 3px solid var(--ax-border-focus);
+    outline-offset: -3px;
   }
-}
 
-a.navds-internalheader__title:active,
-button.navds-internalheader__title:active,
-.navds-internalheader__button:active {
-  background-color: var(--ac-internalheader-active-bg, var(--a-surface-inverted-active));
+  &:active {
+    background: var(--ax-bg-neutral-moderate-pressed);
+  }
 }
 
 .navds-internalheader__user-button {
```

Comment:
Looks like UserButton is not covered by any story. Off topic, but would be nice if you added one (can do it later).

Reply:
Added a story 👍